app/middlewares/user.js

```diff
@@ -0,0 +1,20 @@
+const validator = require('validator');
+const errors = require('../errors');
+
+exports.validateEmail = email => {
+  if (!validator.isEmail(email)) {
+    throw errors.badRequest('No valid email');
```

Comment:
lets use a different syntax for this. I'm thinking about and object that has an internal code, a specific field and a message.